packages/base/src/commands.ts

```diff
@@ -678,6 +691,7 @@ export function addCommands(
     },
     execute: args => {
       const current = tracker.currentWidget;
+      console.log('UNDO PRESSED')
```

Comment:
Just temporarily added these couple of log statements to get help while testing. Will remove them.